web/templates/index.html

```diff
@@ -83,25 +91,29 @@ <h1>botamusique Web Interface</h1>
                 <tbody id="playlist-table" class="playlist-table">
                     <tr id="playlist-loading">
                         <td colspan="4" class="text-center">
-                            <img style="margin: auto; width: 35px;" src="static/image/loading.svg" />
+                            <img style="margin: auto; width: 35px;" src="static/image/loading.svg"
+                                alt="A loading spinner" />
                         </td>
                     </tr>
                     <tr id="playlist-empty" class="d-none">
                         <td colspan="4" class="text-center">
-                            <img style="margin: auto; width: 35px;" src="static/image/empty_box.svg" />
+                            <img style="margin: auto; width: 35px;" src="static/image/empty_box.svg"
+                                alt="A drawing of an empty box." />
                         </td>
                     </tr>
                     <tr class="playlist-expand table-dark d-none">
                         <td colspan="4" class="text-center">
-                            <a class="text-muted" href="javascript:">See item <span class="playlist-expand-item-range"></span> on the playlist.</a>
+                            <a class="text-muted" href="javascript:">See item <span
+                                    class="playlist-expand-item-range"></span> on the playlist.</a>
                         </td>
                     </tr>
                     <tr class="playlist-item-template d-none">
                         <th scope="row" class="playlist-item-index d-none d-md-table-cell"></th>
```

Comment:
@TylerVigario @TerryGeng what does this table header do?

I've seen its content change from nothing, to 1, to 2, etc, when adding songs to the playlist, and down from 3 to 2 and from 2 to 1 when removing songs from the playlist. What is manipulating this header, and what is its purpose? Do you know?

I ask because if it is just like a global variable or something like that, I can mark it as `aria-hidden` for screen reader users.

Reply:
If you have more than 10 items on the playlist (like sometimes I will add about ~100 songs to the playlist), the web interface will only display 10 items of them to avoid making the entire webpage too long.

I'm actually thinking about pagination like what it is like in the library section. I think I will implement it in the next few weeks after I manage to get #158 done.

Reply:
> the web interface will only display 10 items of them to avoid making the entire webpage too long.

Of course :3 that's good.

But what I mean to ask is what does this `<th>` tag do. Is it a counter of the items in the playlist? :)

> I'm actually thinking about pagination like what it is like in the library section

Ohh, actually now that you mention it, that would be sweet. I hadn't realized that they were implemented differently, in fact, I didn't even think to look at the library impl :D

Reply:
I think this `<th>` is displayed as the index of items.
![image](https://user-images.githubusercontent.com/2306637/86439283-df47d100-bd3a-11ea-93b3-e1a3ccda2eb2.png)

Reply:
I thought so too, but its `display` attribute is always `none`. I think it's being cloned every time one adds a song, and the clone's `display` attribute is set to `show`

Reply:
It's the one `<th>` that always has the class `playlist-item-index d-none d-md-table-cell`, even when there are no songs in the playlist :3

Reply:
Oh. You mean this one _which is always hidden_. This is a template, every time a song is being added to the playlist, I clone this object and set its display attribute to `block`.

Reply:
Yaaaaaas!

Reply:
Okay, that means that everything's alright with regards to that header :)

Reply:
Context: [the tool I'm using](https://wave.webaim.org/) was complaining because it wants to see all table headers with text. This one header however, is alright by the accessibility standards, because its `display` is always `none`. This means that the tool is just confused.

This completes the first checklist :)

Reply:
PS: [here](https://webaim.org/techniques/tables/data) is the justification for wanting text in the table headers. It makes sense, it's just that WAVE (the tool) can't see that they are being hidden.